src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -417,6 +507,17 @@ public String getDisplayName() {
       return "Mesos Cloud";
     }
 
+    @Restricted(DoNotUse.class) // stapler only
```

Comment:
We use fully formed sentences for our comments, i.e., we use proper capitalization and periods.

// Stapler only.

Please fix and everywhere else in this patch.